### PR TITLE
Update test-system update docs doing while doing wmde 7&8

### PR DESCRIPTION
**Pull request by addshore**
Last time I ran through this I wrote up docs on how to
create the test systems, but was very light on updating.

Now I have run through an update for wmde7 and wmde8 I
have documented the process.

### CODE REVIEW

File: docs/topics/test-system.md [hunk 1]
```diff
@@ -161,9 +165,44 @@ So Adam will write these docs once that task is merged and resolved.
...
 
-Example for workflow run [1157808966](https://github.com/wmde/wikibase-release-pipeline/actions/runs/1157808966).
+**If the query service updater is restarting**, it is likely due to updates not having happened in the past month.
+
+```sh
+sudo docker-compose -f docker-compose.yml -f docker-compose.extra.yml stop wdqs-updater
+sudo docker-compose -f docker-compose.yml -f docker-compose.extra.yml run --rm wdqs-updater bash
+
+# Within the wdqs-updater shell run the following, with the current date
+/wdqs/runUpdate.sh -h http://"$WDQS_HOST":"$WDQS_PORT" -- --wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --entityNamespaces "120,122" --init --start 20220908000000
```
Comment (manicki) [inline docs/topics/test-system.md:201]: `20220908000000` is the current date I suppose, suggesting to amend the line above for super clarity

File: docs/topics/test-system.md [hunk 2]
```diff
@@ -161,9 +165,44 @@ So Adam will write these docs once that task is merged and resolved.
...
+```
 
-Example for workflow run [1157808966](https://github.com/wmde/wikibase-release-pipeline/actions/runs/1157808966).
+**If the query service updater is restarting**, it is likely due to updates not having happened in the past month.
+
+```sh
+sudo docker-compose -f docker-compose.yml -f docker-compose.extra.yml stop wdqs-updater
+sudo docker-compose -f docker-compose.yml -f docker-compose.extra.yml run --rm wdqs-updater bash
+
+# Within the wdqs-updater shell run the following, with the current date
```
Comment (manicki) [inline docs/topics/test-system.md:200]:
```suggestion
# Within the wdqs-updater shell run the following, with the current date (`20220908000000` in the example line below)
```

File: docs/topics/test-system.md [hunk 3]
```diff
@@ -161,9 +165,44 @@ So Adam will write these docs once that task is merged and resolved.
...
+That would look something like this...
+
+```sh
+TEST_SYSTEM=fedprops-previous
+
+cd /opt/test-systems/$TEST_SYSTEM
+sudo docker cp ${TEST_SYSTEM}_wikibase_1:/var/www/html/LocalSettings.php /tmp/LocalSettings-${TEST_SYSTEM}-$(date --iso).php
+sudo docker-compose -f docker-compose.yml -f docker-compose.extra.yml down
+cd /opt/test-systems
+mv ./$TEST_SYSTEM ./$(date --iso)-${TEST_SYSTEM}
```
Comment (manicki) [inline docs/topics/test-system.md:179]: uber edge case but to avoid differing file/dir names if running this on midnight one would probably put the result `$(date --iso)` into a variable, and reference it here and a few line above?

**Review by manicki — APPROVED** [review]
seems alright at glance

File: docs/topics/test-system.md (hunk 3 above)
Comment (addshore) [reply docs/topics/test-system.md:179]: Sounds like a plan, I'll put it in a draft PR and check it all works next time I run through it!

Comment (addshore) [reply docs/topics/test-system.md:179]: https://github.com/wmde/wikibase-release-pipeline/pull/372
